### PR TITLE
updated or removed dead and outdated links

### PR DESCRIPTION
**Pull request by titusfortner**
Removed links are dead and content is not cached anywhere due to robots.txt on the sites
Alternately, these can be maintained and referenced as [dead-link].
What is the standard for this project?

### CODE REVIEW

File: rb/README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ JRuby and Rubinius.
 ## Links
 
 * http://rubygems.org/gems/selenium-webdriver
-* http://selenium.googlecode.com/git/docs/api/rb/index.html
-* https://github.com/SeleniumHQ/selenium/wiki/Ruby-Bindings
+* http://www.rubydoc.info/gems/selenium-webdriver
+* http://code.google.com/p/selenium/wiki/RubyBindings
```
Comment (ddavison) [inline rb/README.md]: this is invalid.  Selenium no longer uses Google Code wiki, it's all in GitHub

**Comment by titusfortner** [comment]
Ouch, that was a bad rebase from before the 3/12 commit. Thanks for catching it.

File: rb/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ JRuby and Rubinius.
 ## Links
 
 * http://rubygems.org/gems/selenium-webdriver
-* http://selenium.googlecode.com/git/docs/api/rb/index.html
+* http://www.rubydoc.info/gems/selenium-webdriver
```
Comment (lukeis) [inline rb/README.md]: this should be http://seleniumhq.github.io/selenium/docs/api/rb/index.html

**Comment by titusfortner** [comment]
The last update for that was February 17, which means before the current release. Is it still being updated? How/how often?

**Comment by lukeis** [comment]
it should be updated by the person doing the Ruby client bindings release (as apart of the release process). See [the wiki](https://github.com/SeleniumHQ/selenium/wiki/Releasing-Selenium#for-ruby)

**Comment by lukeis** [comment]
must say, the wiki needs updating, since we changed to having the api docs in the gh-pages branch

**Comment by titusfortner** [comment]
Ah, I see, and nothing was changed in ruby section between 2/17 and the release. That makes sense. I'll update that link.

**Comment by titusfortner** [comment]
Pushed updated link

**Comment by andreastt** [comment]
Thanks! Integrated as 6f3c0cf8158b485e84a64d31d4d19b20e4b5a3a6.

**Pull request closed without merging**